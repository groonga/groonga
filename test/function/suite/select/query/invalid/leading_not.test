table_create Entries TABLE_PAT_KEY ShortText
column_create Entries content COLUMN_SCALAR Text

table_create Terms TABLE_PAT_KEY|KEY_NORMALIZE ShortText --default_tokenizer TokenBigram
column_create Terms entries_content_index COLUMN_INDEX|WITH_POSITION Entries content

load --table Entries
[
{"_key":    "The first post!",
 "content": "Welcome! This is my first post!"},
{"_key":    "Groonga",
 "content": "I started to use groonga. It's very fast!"},
{"_key":    "Mroonga",
 "content": "I also started to use mroonga. It's also very fast! Really fast!"},
{"_key":    "Good-bye Senna",
 "content": "I migrated all Senna system!"},
{"_key":    "Good-bye Tritonn",
 "content": "I also migrated all Tritonn system!"}
]

select Entries --match_columns content --query '-fast'
