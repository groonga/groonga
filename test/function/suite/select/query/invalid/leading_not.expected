table_create Entries TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
column_create Entries content COLUMN_SCALAR Text
[[0,0.0,0.0],true]
table_create Terms TABLE_PAT_KEY|KEY_NORMALIZE ShortText --default_tokenizer TokenBigram
[[0,0.0,0.0],true]
column_create Terms entries_content_index COLUMN_INDEX|WITH_POSITION Entries content
[[0,0.0,0.0],true]
load --table Entries
[
{"_key":    "The first post!",
 "content": "Welcome! This is my first post!"},
{"_key":    "Groonga",
 "content": "I started to use groonga. It's very fast!"},
{"_key":    "Mroonga",
 "content": "I also started to use mroonga. It's also very fast! Really fast!"},
{"_key":    "Good-bye Senna",
 "content": "I migrated all Senna system!"},
{"_key":    "Good-bye Tritonn",
 "content": "I also migrated all Tritonn system!"}
]
[[0,0.0,0.0],5]
select Entries --match_columns content --query '-fast'
[[[-63,0.0,0.0],"Syntax error! (-fast)"],[]]
#|e| Syntax error! (-fast)
